plugins/GSdx/GSRendererDX.cpp

```diff
@@ -429,12 +435,10 @@ void GSRendererDX::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sourc
 
 	// vs
 
-	GSDeviceDX::VSSelector vs_sel;
-
-	vs_sel.tme = PRIM->TME;
-	vs_sel.fst = PRIM->FST;
-	vs_sel.logz = dev->HasDepth32() ? 0 : m_logz ? 1 : 0;
-	vs_sel.rtcopy = !!rtcopy;
+	m_vs_sel.tme = PRIM->TME;
+	m_vs_sel.fst = PRIM->FST;
+	m_vs_sel.logz = dev->HasDepth32() ? 0 : m_logz ? 1 : 0;
+	m_vs_sel.rtcopy = !!rtcopy;
```

Comment:
I'm not sure if these sort of boolean type casts on a pointer variable is nice, maybe ``!= nullptr`` check would be much more readable? any opinion? (@refractionpcsx2  @turtleli )

Reply:
I prefer `!= nullptr`.